doc/releases/changelog-dev.md

```diff
@@ -150,6 +150,11 @@
 * `expval` and `var` operations no longer keep the static shots attribute, as a step towards supporting dynamic shots across catalyst.
   [(#1360)](https://github.com/PennyLaneAI/catalyst/pull/1360)
 
+* Installing Catalyst from source via `make frontend` (and similarly for `make all`) now has the
+  option to enable verbose output in the underlying `pip install` command. This option is enabled by
+  appending `VERBOSE=1` to the `make` command.
+  [(#1371)](https://github.com/PennyLaneAI/catalyst/pull/1371)
+
```

Comment:
We don't normally record build system, testing, configuration etc changes since they are not part of the released package.

Reply:
Removed in 1a917b65.